installing/installing-preparing.adoc

```diff
@@ -0,0 +1,86 @@
+
+If you use a user-provisioned installation method, you can configure a proxy for your cluster. The instructions are included in each installation procedure.
+
+If you want to prevent your cluster on a public cloud from exposing endpoints externally, you can deploy a private cluster with installer-provisioned infrastructure on xref:../installing/installing_aws/installing-aws-private.adoc#installing-aws-private[AWS], xref:../installing/installing_azure/installing-azure-private.adoc#installing-azure-private[Azure], or xref:../installing/installing_gcp/installing-gcp-private.adoc#installing-gcp-private[GCP].
+
+You can also configure the cluster machines to use xref:../installing/installing-fips.adoc#installing-fips[FIPS Validated / Modules in Process cryptographic libraries] during installation.
+
+[id="installing-preparing-cluster-for-users"]
+== Preparing your cluster for users
+
+Because you do not have the opportunity to configure all of the supporting infrastructure for your cluster during installation, you need to perform additional configuration between the completion of cluster deployment and when your users access the cluster. You can xref:../installing/install_config/customizations.adoc#customizations[customize] the Operators that make up your cluster and integrate other required systems, such as an identity provider.
+//This link will change when we consolidate the customizations page with the post-installation activities.
+
+For a production cluster, you must configure the following integrations:
+
+* xref:../storage/understanding-persistent-storage.adoc#understanding-persistent-storage[Persistent storage]
+* xref:../authentication/understanding-identity-provider.adoc#understanding-identity-provider[An identity provider]
+* xref:../monitoring/configuring-the-monitoring-stack.adoc#configuring-the-monitoring-stack[Monitoring] for the cluster itself
+* xref:../monitoring/enabling-monitoring-for-user-defined-projects.adoc#enabling-monitoring-for-user-defined-projects[Monitoring] for application workloads
+
+[id="installing-preparing-cluster-for-workloads"]
+== Preparing your cluster for workloads
+
+Depending on your workload needs, you might need to take extra steps before you begin deploying applications. After you prepare infrastructure to support your application xref:../builds/build-strategies.adoc#build-strategies[build strategy], you might need to make provisions for xref:../scalability_and_performance/cnf-performance-addon-operator-for-low-latency-nodes.adoc#cnf-performance-addon-operator-for-low-latency-nodes[low-latency] workloads or xref:../nodes/pods/nodes-pods-secrets.adoc#nodes-pods-secrets[protect sensitive workloads].
```

Comment:
You could add "For example," to the start of the second sentence, to show that it is linked to the first statement and to indicate that there might be other preparations in addition to these. I.e.:

"For example, after you prepare infrastructure to support your application..."